demo

```diff
@@ -7,7 +7,7 @@
 # that aren't allowed.
 DOCKER_COMPOSE_PROJECT_NAME=${DOCKER_COMPOSE_PROJECT_NAME:-$(tr -d ".-" <<<${PWD##*/})}
 
-EXTRA_FILES=
+EXTRA_FILES="-f docker-compose.connect.yml"
```

Comment:
The other default files are specified inline in several places. That's actually quite bad, but I think mixing the two approaches is worse. So I suggest either:

1. Clean up the whole approach, and specify all files here.
2. Or specify this together with the inline files instead, so that we keep them together.

Reply:
I agree with this, sorry. The `EXTRA_FILES` has a clear functionality and we shouldn't mix it.

Reply:
@lluiscampos @kacf ack, reworked as per your suggestions.